### PR TITLE
Sebastianmaj/masteronlybump

### PR DESCRIPTION
**Pull request by SebastianMaj**
**Why**
The Master branch was not creating a new release on version bump which wasn't triggering the PyPi build/publish process on Master

**How**
Added a new Github action that will create a new release on version bump which is triggered by push and checks the tag using regex

**References**
Solves: [Issue #20](https://github.com/NebraLtd/hm-pyhelper/issues/20)

### CODE REVIEW

File: .github/workflows/publish-to-pypi-test.yml [hunk 1]
```diff
@@ -111,8 +109,29 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@master
+    - name: Create release
+      uses: Roang-zero1/github-create-release-action@master
+      with:
+        version_regex: ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Create GitHub release
+      uses: Roang-zero1/github-upload-release-artifacts-action@master
```
Comment (marvinmarnold) [inline .github/workflows/publish-to-pypi-test.yml:131]: This hasn't been updated since 2019, is there something else we can use that is better maintained?

Comment (SebastianMaj) [reply .github/workflows/publish-to-pypi-test.yml:131]: Apparently this one is still widely used. Considering it uses the release api which doesn't really change, it doesn't need active maintenance. I'll look further and see if I can find an actively maintained repo though.

**Review by marvinmarnold — APPROVED** [review]
Could you be sure to squash-and-commit these?

File: .github/workflows/publish-to-pypi-test.yml (hunk 1 above)
Comment (marvinmarnold) [reply .github/workflows/publish-to-pypi-test.yml:131]: Thanks for checking.

**Pull request merged**